src/state_awset_ps.erl

```diff
@@ -102,6 +102,18 @@ new() ->
 new([]) ->
     new().
 
+-spec new_delta() -> delta_state_awset_ps().
+new_delta() ->
+    state_type:new_delta(?TYPE).
```

Comment:
The delta object of the state_awset_ps type cannot be generated from the state object, since the last components are different: a state object: {vclock, []}, a delta object: ordsets:new(). Please check `is_bottom()`.